hash.c

```diff
@@ -4215,32 +4254,25 @@ rb_hash_invert_i(VALUE key, VALUE value, VALUE hash)
 
 /*
  *  call-seq:
- *     hsh.invert -> new_hash
- *
- *  Returns a new hash created by using <i>hsh</i>'s values as keys, and
- *  the keys as values.
- *  If a key with the same value already exists in the <i>hsh</i>, then
- *  the last one defined will be used, the earlier value(s) will be discarded.
- *
- *     h = { "n" => 100, "m" => 100, "y" => 300, "d" => 200, "a" => 0 }
- *     h.invert   #=> {0=>"a", 100=>"m", 200=>"d", 300=>"y"}
+ *    hash.invert -> new_hash
  *
- *  If there is no key with the same value, Hash#invert is involutive.
- *
- *    h = { a: 1, b: 3, c: 4 }
- *    h.invert.invert == h #=> true
- *
- *  The condition, no key with the same value, can be tested by comparing
- *  the size of inverted hash.
+ *  Returns a new \Hash object with the each key-value pair reversed:
```

Comment:
reversed sounds like reversed order.

Maybe needs more clarification, e.g. perhaps using the word "swapped".

Reply:
Fixed as 'inverted', which agrees with #invert.